app/javascript/components/UswdsOverrides/UswdsPagination.tsx

```diff
@@ -48,7 +50,7 @@ const PaginationPage = ({
           unstyled
           data-testid="pagination-page-number"
           className={linkClasses}
-          aria-label={`Page ${page}`}
+          aria-label={label}
```

Comment:
What do you think about pulling the label directly in here? I know it'd be duplicated but It will reduce the dependencies of the `PaginationPage` component.
```tsx
aria-label={`${i18n.t("page")} ${pageNum}`}
```

Reply:
It could be also calculated in here like
```
const label = `${i18n.t("page")} ${pageNum}`
...
aria-label={label}
```

Reply:
That was my initial implementation but we will have to use `i18n` in `PaginationPage` 
`const i18n = useContext(LanguageContext); ` component also. 

It is already being used at `UswdsPagination` component so it will be duplicated.

Reply:
That's fine since contexts are mean to be shared this way. Context's are already shared among all components.

Reply:
sure, let me update.